common/lib/xmodule/xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -2785,12 +2785,6 @@ def test_bulk_course_publish_signal_publish_firing(self, default):
                     self.store.publish(unit.location, self.user_id)
                     signal_handler.send.assert_not_called()
 
-                    self.store.unpublish(unit.location, self.user_id)
-                    signal_handler.send.assert_not_called()
-
-                    self.store.delete_item(unit.location, self.user_id)
-                    signal_handler.send.assert_not_called()
```

Comment:
This assertion didn't exist in mock 1.0.1 so it always returned `True`, but the `unpublish` and `delete_item` signals actually are sent.  This seems to be correct given that the unit is no longer in draft at this point, but willing to listen to arguments to the contrary.